validator/validation/graphics.py

```diff
@@ -155,10 +156,13 @@ def generate_overview_map(validation_run, outfolder, metric, label, values, dc1,
     else:
         if validation_run.reference_configuration.dataset.short_name == globals.ERA5_LAND:
             dy, dx = -0.1, 0.1
+            lats_map = safe_arange(extent[3], extent[2], dy)
+            lons_map = safe_arange(extent[0], extent[1], dx)
         else:
             dy, dx = -0.25, 0.25
-        lats_map = safe_arange(extent[3], extent[2], dy)
-        lons_map = safe_arange(extent[0], extent[1], dx)
+            lats_map = np.arange(extent[3], extent[2], dy)
+            lons_map = np.arange(extent[0], extent[1], dx)
```

Comment:
The duplication of lats_map/lons_map in both branches of the if-else clause looks like a classic tripping hazard for future maintainers of this code. Why safe_arange in one case but np.arange in the other? Ideal place to hide some bugs, is it really necessary?

Reply:
yes, `np.arange` cause problems with the 0.1 resolution but works fine for 0.25, `safe_arange` always works (might be a bit slower), but I agree we should be consistent.

Reply:
So revert this change?

Reply:
yes, not sure why I even changed it...